eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/ImpTest.hs

```diff
@@ -740,6 +775,17 @@ freshKeyHash = do
   addKeyPair kp
   pure $ hashKey kh
 
+giveCoin ::
+  ShelleyEraImp era =>
+  Addr (EraCrypto era) ->
+  Coin ->
```

Comment:
Let's make this function a little bit more general and better named. We have plans on transitioning previous eras to `ImpTest` too. It is possible to create more specific functions out of more general ones, but not the other way around.
```suggestion
sendCoinTo ::
  ShelleyEraImp era =>
  Addr (EraCrypto era) ->
  Coin ->
  ImpTestM era ()
sendCoinTo addr = sendValueTo addr . inject

sendValueTo ::
  ShelleyEraImp era =>
  Addr (EraCrypto era) ->
  Value era ->
```